aten/src/ATen/cuda/CUDAGraph.cpp

```diff
@@ -84,6 +84,7 @@ void CUDAGraph::capture_begin(MempoolId_t pool/*=0*/) {
   capture_gen_ = gen;
   capture_dev_ = c10::cuda::current_device();
 
+  c10::cuda::CUDACachingAllocator::incOngoingCaptures(capture_dev_);
```

Comment:
TODO: update the comment below to reflect that we decided to move the updating of caching allocator state here to avoid a potential race